### PR TITLE
Added appropriate error message on package installation failure

### PR DESCRIPTION
**Pull request by phanidevavarapu**
Added appropriate error message on failure during the installation/upgrade of a package.

### CODE REVIEW

File: client/internal/packagessyncer.go [hunk 1]
```diff
@@ -278,7 +278,7 @@ func (s *packagesSyncer) downloadFile(ctx context.Context, pkgName string, file
 
 	err = s.localState.UpdateContent(ctx, pkgName, resp.Body, file.ContentHash)
 	if err != nil {
-		return fmt.Errorf("cannot download file from %s: %v", file.DownloadUrl, err)
+		return fmt.Errorf("failed to install/update the package %s: %v", pkgName, err)
```
Comment (tigrannajaryan) [inline client/internal/packagessyncer.go:281]: I don't think the error message here should be changed like this. The error is in downloading the file and that's what it should say. If you want to have additional information like the package name it is fine to include it. Alternatively you can enrich the error message at the call site and add the package name.

Comment (phanidevavarapu) [reply client/internal/packagessyncer.go:281]: But if we look at the actual code flow, we are past the downloading the package.

Comment (tigrannajaryan) [reply client/internal/packagessyncer.go:281]: You are right. Can we include both package name and URL we downloaded from in the message? Something like this: `failed to install/update the package %s downloaded from %v`?

**Review by tigrannajaryan — APPROVED** [review]

**Comment by tigrannajaryan** [comment]
Thank you @phanidevavarapu

**Pull request merged**